src/main/resources/interface-specification.yml

```
@@ -7231,6 +7305,10 @@ components:
         - id
         - state
         - dailyCalls
+        - expectedApprovalDate
+        - riskAnalysisDocument
+        - createdAt
+        - firstActivationAt
```

those fields are optional
```suggestion
        - createdAt
```